prod-devel-rcar.yaml

```diff
@@ -189,7 +189,7 @@ components:
       - *COMMON_SOURCES
       - type: git
         url: https://github.com/xen-troops/meta-renesas.git
-        rev: "Renesas-Yocto-v5.1-patched"
+        rev: "Renesas-Yocto-v5.9-patched"
```

Comment:
So Renesas still provides separate patches to own meta-layers?

Reply:
As far I know there are no extra patches needed. The branch here is for another reason, we need to temporarily revert some patches to make pvrkm/pvrum versions we use happy. But, as it was already discussed internally, it would be better, of course, to avoid creating any extra branches and try resolve that internally (I mean within meta-xt-* layers).